python/metatensor-operations/tests/contiguous.py

```diff
@@ -0,0 +1,71 @@
+import os
+
+import metatensor
+from metatensor import TensorBlock, TensorMap
+
+DATA_ROOT = os.path.join(os.path.dirname(__file__), "data")
+
+tensor = metatensor.load(
+        os.path.join(DATA_ROOT, "qm7-power-spectrum.npz"),
+        use_numpy=True,
+    )
```

Comment:
This should either be a global variable, or a function. Perhaps in this case it best we make it into a pytest fixture so it can be used in all the unit tests in this file. It will look like this, but we can discuss how to use it if you like - see https://docs.pytest.org/en/7.1.x/how-to/fixtures.html

```suggestion
@pytest.fixture
def tensor():
    """Loads a TensorMap from file for use in tests"""
    return metatensor.load(
        os.path.join(DATA_ROOT, "qm7-power-spectrum.npz"),
        use_numpy=True,
    )
```